lib/vsc/install/shared_setup.py

```diff
@@ -1158,11 +1158,6 @@ def cleanup(self, prefix=''):
                 except OSError:
                     log.error("cleanup failed for %s" % d)
 
-        for fn in ('setup.cfg',):
-            ffn = prefix + fn
-            if os.path.isfile(ffn):
-                os.remove(ffn)
```

Comment:
@JensTimmerman since it's here so explicitly, there must be a reason for it?

Have you tried figuring out why it was added?

Also, what made you make this change? Why is this a problem all of a sudden?